examples/llama3-fine-tuning-lora/llama3_fine_tuning.py

```diff
@@ -266,12 +262,10 @@ def generate(self, query: str, max_length: int = DEFAULT_MAX_LENGTH):
         image=img,
         provider="aws",
     ).up_if_not()
-
-    cluster.sync_secrets(["huggingface"])
-
+    cluster.restart_server()
     # Finally, we define our module and run it on the remote cluster. We construct it normally and then call
     # `to` to run it on the remote cluster. Alternatively, we could first check for an existing instance on the cluster
-    # by calling `cluster.get(name="llama3-medical-model")`. This would return the remote model after an initial run.
+    # by calling `cluster.get(name="llama3-medical-model", remote = True)`. This would return the remote model after an initial run.
```

Comment:
```suggestion
    # by calling `cluster.get(name="llama3-medical-model", remote=True)`. This would return the remote model after an initial run.
```